spec/models/game_session_spec.rb

```diff
@@ -1,28 +1,91 @@
 require 'rails_helper'
 require 'support/custom_matchers'
 
+RSpec.describe 'GameSession model class constants' do
```

Comment:
This is really great!

Just fyi, whether you use `RSpec.describe` or the helper `describe` is a matter of opinion. You can see a similar discussion here about FactoryBot methods: https://marketplacer.slack.com/archives/C5P5N1BUM/p1657684526163439

What I would recommend is wrapping the entire model test. Here's two examples from our codebase:

```ruby
RSpec.describe Brand, type: :model do
  describe '#valid?' do
    it 'is valid with valid_attributes' do
      expect(subject).to be_valid
    end
  end
end
```

```ruby
describe Account do
  describe 'included modules' do
    it 'includes AccountPublicLanguage module' do
      expect(described_class.ancestors).to include(AccountPublicLanguage)
    end
  end
end
```

See https://relishapp.com/rspec/rspec-rails/v/5-1/docs/model-specs

Reply:
Then within that overall model spec block you'll have:

describe 'GameSession model class constants'
describe '.create'
context '#lives_remaining'

By convention, class methods are decribed with a `.` and instance methods with a `#`

https://www.betterspecs.org/#describe